### PR TITLE
Fixed spelling error in issue template

### PR DESCRIPTION
**Pull request by Kshitiz-Mhto**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Comment by CalvinWilkinson** [comment]
@Kshitiz-Mhto Great work on the branch name.  😀  I appreciate that. My CI/CD works off of this stuff.

**Comment by CalvinWilkinson** [comment]
@Kshitiz-Mhto Could you please use the PR template for the PR?

You can find it in the location of the repo here 👉🏼`~/.github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md`
Just copy the contents of the markdown file into the main description above.

I have different kinds PR templates depending on what branch is moving to another.
Once the PR template is added, I will finish code reviewing it.  Thanks!!

File: .github/ISSUE_TEMPLATE/bug-issue-template.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ body:
 - type: checkboxes
   attributes:
     label: Operating System
-    description: Which operating system have you seen the bug occur on
+    description: Which operating system have you seen the bug occurred on?
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/bug-issue-template.yml:51]: Please refactor to the suggestion below

```suggestion
    description: Which operating system have you seen the bug occur on?
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Comment by KinsonDigitalAdmin** [comment]
@Kshitiz-Mhto Thanks so much for your help!!

I have reviewed the template and it is good to go.   I did not notice one thing though.  The target branch is not set correctly.
Below are some highlights/info about the **source** and **target** branches and what they should be.

image

---

I went ahead and change the branch already.  I am in the middle of updating the branching documentation to contain info about how to deal with **preview feature branches**. 😀

Just in case your curious, the issue below is the issue I just finished in regard to the branching setup for this project.
- #146 

Also, [here](https://github.com/KinsonDigital/Velaptor/blob/preview/v1.0.0-preview.10/Documentation/Branching.md) is some info on branching.

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Pull request merged**